### ISSUE TITLE
[Bug]: Accessibility - Toggle menu button doesn't appear to toggle the menu

### ISSUE THREAD

**Issue by Almost-Senseless-Coder** [bulwarkmail/webmail#721]
### Description

Whilst the `Toggle menu` button has the `aria-expanded` role, signaling when the menu should be revealed or hidden, the menu itself doesn't appear anywhere in the UI from my point of view.

### Steps to Reproduce

1. Activate any screen reader
2. Click the `Toggle menu` button
3. The reported state should shift from `collapsed` to `expanded` (and vise versa), but the menu doesn't appear anywhere in the UI where I can find it\*.

\* Themenu is actually visible all the time, see my comment below. To fix this, add `aria-hidden="true"` when the menu is collapsed, or `display: none`.

### Expected Behavior

The menu should appear somewhere in the UI. Ideally, focus would be moved to the first item of the menu upon toggling the menu state to expanded. Next best thing would be making the menu appear right underneath the `Toggle menu` button.

### Actual Behavior

As near as I could tell, the menu doesn't appear anywhere in the UI--even though, according to the menu button state, it should be expanded.

### Bulwark Version

1.7.8

### Stalwart Mail Server Version

0.16.15

### Operating System

None

### Relevant Logs or Error Output

```shell

```

**Comment by Almost-Senseless-Coder** [comment]
Turns out the menu the `Toggle menu` button is supposed to toggle is always visible to screen readers, i.e., the hiding of the relevant UI elements only happens visually.

This is particularly problematic because actions like the account switcher only trigger if the menu is expanded, leading to a confusing user experience for screen reader users.
